src/Parser/Hunter/Marksmanship/Modules/Items/Tier20_2p.js

```diff
@@ -4,26 +4,37 @@ import Combatants from 'Parser/Core/Modules/Combatants';
 import SPELLS from 'common/SPELLS';
 import SpellIcon from 'common/SpellIcon';
 import SpellLink from 'common/SpellLink';
-import { formatPercentage } from "common/format";
+import HIT_TYPES from "Parser/Core/HIT_TYPES";
+import { formatPercentage, formatNumber} from "common/format";
+import getDamageBonus from "../../../Shared/Core/getDamageBonus";
+
+const T20_2P_CRIT_DMG_BONUS = 0.15;
 
 class Tier20_2p extends Module {
   static dependencies = {
     combatants: Combatants,
   };
   totalAimed = 0;
   buffedAimed = 0;
+  bonusDmg = 0;
 
   on_initialized() {
     this.active = this.combatants.selected.hasBuff(SPELLS.HUNTER_MM_T20_2P_BONUS.id);
   }
   on_byPlayer_damage(event) {
-    if(event.ability.guid !== SPELLS.AIMED_SHOT.id) {
-      return;
+    const spellId = event.ability.guid;
+    const isCrit = event.hitType === HIT_TYPES.CRIT;
+    if (this.combatants.selected.hasBuff(SPELLS.HUNTER_MM_T20_2P_BONUS_BUFF.id)) {
```

Comment:
I assume `HUNTER_MM_T20_2P_BONUS` is for the bonus the 2p actually gives while `..._BONUS_BUFF` is the buff it gives when it procs or something. In that case, there should be a second parameter in the `hasBuff`, so `hasBuff(id, event.timestamp)`.